### PR TITLE
ADEN-5104 add src=premium for slots on pages with premium videos

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-5104

travis fails not connected with the PR code

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [helper.js#L18](https://github.com/Wikia/app/blob/1f10589ea547b9e1fa634040e38050d8c4e7d49d/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L18): Function has 14 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [helper.js#L57](https://github.com/Wikia/app/blob/1f10589ea547b9e1fa634040e38050d8c4e7d49d/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L57): Function has a complexity of 17 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AdEngine/js/provider/gpt/helper.js [hunk 1]
```diff
@@ -96,16 +96,18 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 		}
 
 		function setAdditionalTargeting(slotTargetingData) {
-			if (adShouldBeRecovered) {
+			if (adContext.getContext().targeting.hasFeaturedVideo) {
+				slotTargetingData.src = 'premium';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js]: What about evolve/turtle providers?

**Review by fraszczakszymon — CHANGES_REQUESTED** [review]
What about evolve/turtle providers?

**Review by v3rth — COMMENTED** [review]
LGTM

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**